baked_in.go

```diff
@@ -229,47 +229,47 @@ func parseOneOfParam2(s string) []string {
 	return vals
 }
 
-func isURLEncoded(fl FieldLevel) bool {
+func isURLEncoded(v *Validate, fl FieldLevel) bool {
 	return uRLEncodedRegex.MatchString(fl.Field().String())
 }
 
-func isHTMLEncoded(fl FieldLevel) bool {
+func isHTMLEncoded(v *Validate, fl FieldLevel) bool {
 	return hTMLEncodedRegex.MatchString(fl.Field().String())
 }
 
-func isHTML(fl FieldLevel) bool {
+func isHTML(v *Validate, fl FieldLevel) bool {
 	return hTMLRegex.MatchString(fl.Field().String())
 }
 
-func isOneOf(fl FieldLevel) bool {
+func isOneOf(v *Validate, fl FieldLevel) bool {
 	vals := parseOneOfParam2(fl.Param())
 
 	field := fl.Field()
 
-	var v string
+	var val string
 	switch field.Kind() {
 	case reflect.String:
-		v = field.String()
+		val = field.String()
 	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
-		v = strconv.FormatInt(field.Int(), 10)
+		val = strconv.FormatInt(field.Int(), 10)
 	case reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64:
-		v = strconv.FormatUint(field.Uint(), 10)
+		val = strconv.FormatUint(field.Uint(), 10)
 	default:
 		panic(fmt.Sprintf("Bad field type %T", field.Interface()))
 	}
 	for i := 0; i < len(vals); i++ {
-		if vals[i] == v {
+		if vals[i] == val {
 			return true
 		}
 	}
 	return false
 }
 
 // isUnique is the validation function for validating if each array|slice|map value is unique
-func isUnique(fl FieldLevel) bool {
+func isUnique(v *Validate, fl FieldLevel) bool {
 	field := fl.Field()
 	param := fl.Param()
-	v := reflect.ValueOf(struct{}{})
+	val := reflect.ValueOf(struct{}{})
```

Comment:
Is this rename necessary?

```suggestion
	v := reflect.ValueOf(struct{}{})
```

Reply:
Yes because of the new input parameter `v *Validate`. That parameter is needed to meet the function signature defined at https://github.com/letsencrypt/validator/pull/4/files#diff-6eda4809e29e9c01c780a30c46947d23b10606dbd948fd00dc5376030cf897c6R22